css-scroll-anchoring-1/Overview.bs

```diff
@@ -85,16 +86,41 @@ An anchor node can be any <a>box</a> except one for a non-<a>atomic inline</a>.
 The anchor node is always a <a>descendant</a> of the <a>scrolling box</a>.
 In some cases, a scrolling box may not select any anchor node.
 
+An element |C| is a <dfn id="anchor-viable-candidate">viable candidate</dfn>
+for becoming a scroll anchor for a scrolling box |S| if it meets all of the
+following criteria:
+  * |C| is a non-<a>atomic inline</a>.
+  * |C| is <a>partially visible</a> or <a>fully visible</a> in |S|
+  * |C| is a descendant of |S|
+  * |C| is not in an <a>excluded subtree</a>
+  * None of the ancestors of |C| up to |S| are in an <a>excluded subtree</a>
+
+Some elements are considered to be <dfn
+id="anchor-priority-candidates">priority candidates</dfn> for anchor selection:
+  1. An element that <a href="https://html.spec.whatwg.org/multipage/semantics-other.html#selector-focus">has focus</a>.
```

Comment:
Are these priorities sorted? As in, should we always prefer the focused element over the caret? Or find-in-page?

Reply:
Yes, in the algorithm below I say
"2. Otherwise, for each priority candidate _PC_ **in order specified**, ..."

In other words this block here is listing out the priority candidates, and at use time in the algorithm, we're specifying that the order matters.

Does that make sense or do you think we should reference the importance of the order here as well?

Reply:
No, makes sense :)

Reply:
I think this `#selector-focus` is not the right definition to link to. In particular, per spec it should be true for shadow hosts that contain focused elements. I think it should only choose the deepest focused node.

Reply:
Yeah makes sense, I've updated it to reference https://html.spec.whatwg.org/multipage/interaction.html#dom-document-activeelement instead which I think is exactly what we're looking for here.